src/lib/openapi/spec/project-overview-schema.ts

```diff
@@ -17,42 +17,60 @@ export const projectOverviewSchema = {
     properties: {
         stats: {
             $ref: '#/components/schemas/projectStatsSchema',
+            description: 'Project statistics',
         },
         version: {
             type: 'number',
+            example: 1,
+            description: '',
         },
         name: {
             type: 'string',
+            example: 'dx-squad',
+            description: 'Name of this project used to identify it',
         },
         description: {
             type: 'string',
             nullable: true,
+            example: 'DX squad feature release',
+            description: 'Additional information about the project',
         },
         members: {
             type: 'number',
+            example: 4,
+            description: 'Count of the project members',
         },
         health: {
             type: 'number',
+            example: 50,
+            description: 'Indicator of the project health in the 0-100 scale',
         },
         environments: {
             type: 'array',
             items: {
                 type: 'string',
             },
+            example: ['development', 'production'],
+            description: 'Enabled environments for this project',
         },
         features: {
             type: 'array',
             items: {
                 $ref: '#/components/schemas/featureSchema',
             },
+            description: 'List of features in this project',
```

Comment:
Is it with or without archived features? We should clarify that.

```suggestion
            description: 'The full list of features in this project (excluding archived features)',
```

Reply:
after some experimentation it looks like it's without the archived